packages/PyTrilinos/CMakeLists.txt

```diff
@@ -144,14 +144,14 @@ IF(PyTrilinos_ENABLE_NOX)
   APPEND_SET(PyTrilinos_MODULES  NOX.Abstract  )
   APPEND_SET(PyTrilinos_MODULES  NOX.StatusTest)
   APPEND_SET(PyTrilinos_MODULES  NOX.Solver    )
-  IF(NOX_ENABLE_Epetra)
+  IF(NOX_ENABLE_ABSTRACT_IMPLEMENTATION_EPETRA)
     APPEND_SET(PyTrilinos_MODULES NOX.Epetra.__init__ )
     APPEND_SET(PyTrilinos_MODULES NOX.Epetra.Interface)
-  ENDIF(NOX_ENABLE_Epetra)
-  IF(NOX_ENABLE_PETSC)
+  ENDIF(NOX_ENABLE_ABSTRACT_IMPLEMENTATION_EPETRA)
+  IF(NOX_ENABLE_ABSTRACT_IMPLEMENTATION_PETSC)
```

Comment:
Does anyone actually test the PETSc bits in NOX?

Reply:
@rppawlo ?

Reply:
It might get exercised through xSDK sometimes -- I'm not sure if xSDK has regular testing.